repo/packages/H/hdfs/0/command.json

```diff
@@ -0,0 +1,5 @@
+{
+  "pip": [
+    "git+https://github.com/mesosphere/dcos-hdfs.git#dcos-hdfs=0.1.0"
```

Comment:
what is the `0.1.0` point to though? There is no branch or tag in the git repo to provide any visibility into what is in that "release".